website/static/js/code-block-buttons.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+window.addEventListener("load", () => {
+  function button(label, ariaLabel, icon, className) {
+    const btn = document.createElement("button");
+    btn.classList.add("btnIcon", className);
+    btn.setAttribute("type", "button");
+    btn.setAttribute("aria-label", ariaLabel);
+    btn.innerHTML =
+      '<div class="btnIcon__body">' +
+      icon +
+      '<strong class="btnIcon__label">' +
+      label +
+      "</strong>" +
+      "</div>";
+    return btn;
+  }
```

Comment:
I think the `<div>` is useless here. We can put svg into css (use `::before` or `background-image`), and use `<button>Copy</button>` directly here.